File: findOcurrences/findOcurrences.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# @Time : 2019/6/28 10:25
# @Author : hhx06
# @Site : 
# @File : findOcurrences.py
# @Software: PyCharm
class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        text = text.split(" ")
        new_data =[]
        for i in range (len(text)-2):
            if text[i] == first and text[i+1] ==second:
                new_data.append(text[i+2])
        return new_data

File: findOcurrences/findOcurrences.php
<?php
/**
 * Created by PhpStorm
 * User : Hhx
 * Date : 2019/6/28
 * Time : 9:56
 */
class Solution {

    /**
     * @param String $text
     * @param String $first
     * @param String $second
     * @return String[]
     */
    function findOcurrences($text, $first, $second) {
        // $arr = explode(" ",$text);
        // $len = count($arr)-1;
        // $data = [];
        // for($i =1;$i<$len;$i++){
        //     if($arr[$i] ==$second){
        //         if($arr[$i-1]==$first){
        //             $data[] = $arr[$i+1];
        //         }
        //     }
        // }
        // return $data;

        $arr = explode(" ",$text);
        $len = count($arr)-2;
        $data = [];
        for($i =0;$i<$len;$i++){
            if($arr[$i] ==$first){
                if($arr[$i+1]==$second){
                    $data[] = $arr[$i+2];
                }
            }
        }
        return $data;

    }
}